.gitignore

```diff
@@ -1,4 +1,4 @@
-/node_modules
+/**/node_modules
```

Comment:
This may lead to trouble. We *do* check in some code that happens to live inside folders named node_modules. This will make it confusing if somebody tries to add a new file there.

See for example https://github.com/cardstack/cardstack/blob/master/tests/stub-middleware/node_modules/stub-middleware-extra/middleware.js

The lint errors that I think motivated this change:

```
/home/travis/build/cardstack/cardstack/tests/ephemeral-test-app/node_modules/@cardstack/hub/code-generators.d.ts
  2:1  error  Expected indentation of 2 spaces but found 4  @typescript-eslint/indent
```

are happening because we have a rule to ignore `/packages/hub/code-generators.d.ts` but we don't have a rule to ignore `/tests/ephemeral-test-app/node_modules/@cardstack/hub/code-generators.d.ts`, which is really the same file, but via a symlink. 

Probably we could ignore everything below the symlink like:

```
/tests/ephemeral-test-app/node_modules/@cardstack/hub/**/*
```

Reply:
Makes sense. I removed the `**` and replaced it with specific rules.